compiler-rt/lib/scudo/standalone/stack_depot.h

```diff
@@ -112,7 +112,7 @@ class alignas(atomic_u64) StackDepot {
     if (TabMask == 0)
       return false;
     uptr TabSize = TabMask + 1;
-    if (!isPowerOfTwo(TabSize))
+    if (TabSize == 0 || !isPowerOfTwo(TabSize))
       return false;
```

Comment:
I'm wondering if we should fix `isPowerOfTwo` instead? Zero is not a power of 2.

Reply:
It might be worth doing that, but I do worry that somewhere, something depends on the behavior.

I'll file a bug because I think that's a bigger change and requires some extra verification.

Reply:
I did a quick review and it seems to be fine (there's no additional zero check will be introduced). And yes, it's fine to track it later